test/integration/fixtures/blocks/core__quote__deprecated-2.serialized.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:quote {"align":"right"} -->
-<blockquote class="wp-block-quote has-text-align-right"><!-- wp:paragraph -->
+<blockquote class="wp-block-quote alignright"><!-- wp:paragraph -->
```

Comment:
This changes the output of the deprecation, which means that for a user who opens up this older markup, what once was a setting to align text to the right now sets the overall block to be aligned right.

Reply:
This was not a manual change, I ran the script to update the fixtures - so the error is in the code for the deprecation, correct?

Reply:
> so the error is in the code for the deprecation, correct?

Yes, if we make an update to the deprecations code that results in having to regenerate the serialized version, then it's a good signal that we might have to make a change to the logic in the `deprecated.js` file. For this case, a good indication that the deprecation code is working correctly is if the existing `core__quote__deprecated-2.serialized.html` markup doesn't require any changes to be made in this PR. Ideally, the output of all existing deprecations should remain unchanged.